plaid/api_test.go

```diff
@@ -12,9 +12,9 @@ var (
 	testPublicKey = os.Getenv("PLAID_PUBLIC_KEY")
 	testEnv       = Sandbox
 
-	sandboxInstitution     = "ins_109508"
-	sandboxInstitutionName = "First Platypus Bank"
-	testProducts           = []string{"auth", "identity", "income", "transactions"}
+	sandboxInstitution      = "ins_109508"
+	sandboxInstitutionQuery = "Platypus"
```

Comment:
This is only used by `TestSearchInstitutions`. This matches both `First Platypus Bank` (`ins_109508`) and `Flexible Platypus Open Banking` (`ins_117181`). The former is a mock US institution with `oauth` set to `false`. The latter is a mock UK institution with `oauth` set to `true`.